### PR TITLE
[Doc] correct ceph migration steps

### PR DESCRIPTION
**Pull request by katarimanojk**
update/correct few ceph rbd migration steps
JIRA: https://issues.redhat.com/browse/OSPRH-11496

### CODE REVIEW

**Comment by katarimanojk** [comment]
@fmount This patch concludes the changes in ceph migration doc based on manual verification of every step.

File: docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc [hunk 1]
```diff
@@ -53,10 +53,10 @@ $ sudo nft list ruleset | grep ceph_mgr
 label to the target node:
 +
 ----
-$ ceph orch host label add <target_node> mgr; done
+$ sudo cephadm shell -- ceph orch host label add <target_node> mgr
```
Comment (fmount) [inline docs_user/modules/proc_migrating-mgr-from-controller-nodes.adoc:56]: Thank you

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc [hunk 2]
```diff
@@ -19,7 +19,7 @@ $ sudo cp -R /etc/ceph $HOME/ceph_client_backup
 $ sudo cephadm shell -- ceph mgr stat
 ----
 
-. Fail the `ceph-mgr` if it is active on the source node or target node:
+. Fail the `ceph-mgr` if it is active on the source or draining node :
```
Comment (fmount) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:22]: can we simply remove `or target node` here? I wouldn't add `or draining node`

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc [hunk 3]
```diff
@@ -31,28 +31,28 @@ $ sudo cephadm shell -- ceph mgr fail <mgr_instance>
...
 $ for label in mon mgr _admin; do
-    sudo cephadm shell -- ceph orch host rm label <source_node> $label;
+    sudo cephadm shell -- ceph orch host label rm <source_node> $label;
 done
 ----
 +
 * Replace `<source_node>` with the hostname of the source node.
 
-. Remove the running Ceph Monitor daemon from the source node:
+. (Optional) Ensure to remove Ceph Monitor daemon from the source node if it is still running ):
```
Comment (fmount) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:40]: Remove `)` at the end

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc [hunk 4]
```diff
@@ -11,12 +11,19 @@ IP address migration assumes that the target nodes are originally deployed by
...
 .Procedure
 
-. Get the original Ceph Monitor IP address from the existing `/etc/ceph/ceph.conf` file on the `mon_host` line, for example:
+. Get the original Ceph Monitor IP addresses from `$HOME/ceph_client_backup/ceph.conf` file on the `mon_host` line, for example:
 +
 ----
 mon_host = [v2:172.17.3.60:3300/0,v1:172.17.3.60:6789/0] [v2:172.17.3.29:3300/0,v1:172.17.3.29:6789/0] [v2:172.17.3.53:3300/0,v1:172.17.3.53:6789/0]
 ----
 
+. Match it with storage newtork IP addresses on the source node and find the Ceph Monitor IP address
```
Comment (fmount) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc:20]: instead if `it`, we should explicitly say: `the ip address retrieved in the previous step`. @klgill to give the right direction here.

**Review by fmount — COMMENTED** [review]
Thank you @katarimanojk for these fixes. I did a first pass but let's have a review from @klgill !

**Comment by fmount** [comment]
> update/correct few ceph rbd migration steps

Do you want to link Jira?

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc (hunk 2 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:22]: done

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc (hunk 4 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc:20]: done, @klgill can review the same and edit the statement if required.

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc (hunk 3 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:40]: done

**Comment by fmount** [comment]
I'm still ok w/ this, I'll wait for @klgill to ack/nack

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc [hunk 5]
```diff
@@ -19,7 +19,7 @@ $ sudo cp -R /etc/ceph $HOME/ceph_client_backup
 $ sudo cephadm shell -- ceph mgr stat
 ----
 
-. Fail the `ceph-mgr` if it is active on the source node or target node:
+. Fail the `ceph-mgr` if it is active on the source :
```
Comment (klgill) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:22]:
```suggestion
. Fail the `ceph-mgr` if it is active on the source:
```

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc [hunk 6]
```diff
@@ -31,28 +31,28 @@ $ sudo cephadm shell -- ceph mgr fail <mgr_instance>
...
 $ for label in mon mgr _admin; do
-    sudo cephadm shell -- ceph orch host rm label <source_node> $label;
+    sudo cephadm shell -- ceph orch host label rm <source_node> $label;
 done
 ----
 +
 * Replace `<source_node>` with the hostname of the source node.
 
-. Remove the running Ceph Monitor daemon from the source node:
+. (Optional) Ensure to remove Ceph Monitor daemon from the source node if it is still running :
```
Comment (klgill) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:40]:
```suggestion
. Optional: Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:
```

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc [hunk 7]
```diff
@@ -11,12 +11,19 @@ IP address migration assumes that the target nodes are originally deployed by
...
 
-. Get the original Ceph Monitor IP address from the existing `/etc/ceph/ceph.conf` file on the `mon_host` line, for example:
+. Get the original Ceph Monitor IP addresses from `$HOME/ceph_client_backup/ceph.conf` file on the `mon_host` line, for example:
 +
 ----
 mon_host = [v2:172.17.3.60:3300/0,v1:172.17.3.60:6789/0] [v2:172.17.3.29:3300/0,v1:172.17.3.29:6789/0] [v2:172.17.3.53:3300/0,v1:172.17.3.53:6789/0]
 ----
 
+. Match the ip address retrieved in the previous step with storage newtork IP addresses on the source node and find the Ceph Monitor IP address.
+----
```
Comment (klgill) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc:21]:
```suggestion
. Match the IP address retrieved in the previous step with the storage network IP addresses on the source node, and find the Ceph Monitor IP address:
+
----
```

**Review by klgill — CHANGES_REQUESTED** [review]
I have a few suggested changes.

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc (hunk 6 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:40]: done

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc (hunk 5 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:22]: done

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc (hunk 7 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc:21]: done

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc [hunk 8]
```diff
@@ -31,28 +31,28 @@ $ sudo cephadm shell -- ceph mgr fail <mgr_instance>
...
 $ for label in mon mgr _admin; do
-    sudo cephadm shell -- ceph orch host rm label <source_node> $label;
+    sudo cephadm shell -- ceph orch host label rm <source_node> $label;
 done
 ----
 +
 * Replace `<source_node>` with the hostname of the source node.
 
-. Remove the running Ceph Monitor daemon from the source node:
+. (Optional) Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:
```
Comment (klgill) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:40]:
```suggestion
. Optional: Ensure that you remove the Ceph Monitor daemon from the source node if it is still running:
```
Our style for optional steps is "Optional:", not "(Optional)."

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc [hunk 9]
```diff
@@ -11,12 +11,19 @@ IP address migration assumes that the target nodes are originally deployed by
...
 
-. Get the original Ceph Monitor IP address from the existing `/etc/ceph/ceph.conf` file on the `mon_host` line, for example:
+. Get the original Ceph Monitor IP addresses from `$HOME/ceph_client_backup/ceph.conf` file on the `mon_host` line, for example:
 +
 ----
 mon_host = [v2:172.17.3.60:3300/0,v1:172.17.3.60:6789/0] [v2:172.17.3.29:3300/0,v1:172.17.3.29:6789/0] [v2:172.17.3.53:3300/0,v1:172.17.3.53:6789/0]
 ----
 
+. Match the IP address retrieved in the previous step with the storage network IP addresses on the source node, and find the Ceph Monitor IP address:
+----
```
Comment (klgill) [inline docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc:21]:
```suggestion
. Match the IP address retrieved in the previous step with the storage network IP addresses on the source node, and find the Ceph Monitor IP address:
+
----
```
The plus sign is needed between the step and code for alignment.

**Review by klgill — CHANGES_REQUESTED** [review]
Minor style changes. After these are implemented , this PR LGTM.

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc (hunk 8 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-drain-host.adoc:40]: done

File: docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc (hunk 9 above)
Comment (katarimanojk) [reply docs_user/modules/proc_migrating-mon-from-controller-nodes-network.adoc:21]: done

**Review by klgill — APPROVED** [review]

**Review by fmount — APPROVED** [review]
/lgtm

**Review by jistr — APPROVED** [review]

**Pull request merged**